HW02_B_Mimic.md

```diff
@@ -87,3 +117,18 @@ graph below, I used Set2.
 Now using the above mpg dataset, make this graph
 
 ![](HW02_B_Mimic_files/figure-gfm/unnamed-chunk-8-1.png)<!-- -->
+
+```{r graph 4 code}
+ggplot(mpg,aes(class,cty)) +

```

Comment:
I was able to get a similar figure using (cty, class) in the reverse order to what you have written and then not have the additino of coord_flip().  I am not exactly sure why this worked, but I when looking more closely at the figures the points on your figure look more accurate (closer to the goal figure) than mine so that could have something to do with it.

Reply:
At the beginning I was trying to directly draw a boxplot in a horizontal way, but it didn't work. After I did some research on it, I realized that maybe "boxplot" can only be successfully plotted when the numeric vector is on the y axis. So it seems that the only proper way of doing horizontal boxplot is using the coord_flip() command?